src/renderer/services/deposit/deposit-tx.ts

```diff
@@ -1,9 +1,11 @@
 import {bytes, DepositData} from "@chainsafe/eth2.0-types";
+import {config} from "@chainsafe/eth2.0-config/lib/presets/mainnet";
```

Comment:
didn't see that in original implementation. Config should be param so we can support testnets